### PR TITLE
Add topics to House as per #793.

### LINKED ISSUE #793: house: add topics
As part of the update to the exercism website, we want to add topics to each exercise. This should be topics that will most likely be relevant to the exercise even though you might be able to solve it without encountering that topic. There is a [list of topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) used across all exercism tracks but you can add track specific ones as well if needed.

See issue #523 for a discussion about adding topics. There are also some exercises (e.g. hello-world and two-fer) that already have topics so you can look at them as examples. The topics need to be added to the entry for the relevant exercise in `config.json`.

### PR DESCRIPTION
**Pull request by sjwarner-bp**
Add topics to the House exercise in config.json, for the future version of exercism.

This fixes #793 .

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

**Comment by sjwarner-bp** [comment]
I've also noticed a couple more open issues regarding topics. While they currently seem to be assigned, should the contributors turn out to be inactive, I'd be happy to do those too :slightly_smiling_face:

File: config.json [hunk 1]
```diff
@@ -602,7 +602,11 @@
       "core": false,
       "difficulty": 6,
       "slug": "house",
-      "topics": null,
+      "topics": [
+        "arrays",
+        "loops",
+        "strings"
```
Comment (stkent) [inline config.json]: Maybe also `conditionals`?

**Comment by stkent** [comment]
This commit seems to include `food-chain` changes too? Not sure if those are meant to be in here 🙊

**Comment by sjwarner** [comment]
Oops, you're right! I seem to have added those three extra topics to `food-chain` - I'll move them all over to `house`! Not sure how I managed that...

File: config.json [hunk 2]
```diff
@@ -602,7 +602,14 @@
...
       "difficulty": 6,
       "slug": "house",
-      "topics": null,
+      "topics": [
+        "arrays",
+        "conditionals",
+        "loops",
+        "strings",
+        "text_formatting",
+        "variables"
```
Comment (FridaTveit) [inline config.json]: Great suggestions :) The only one I'm not sure about is `variables`. How do you think that relates to this exercise? You'd have to use variables to solve pretty much every exercise so I would only add it as a topic if using variables was in some way particularly important to the exercise. What do you think? :)

**Comment by sjwarner-bp** [comment]
You're right - I thought it was a bit odd when I looked through the list of all tags, but stuck it in anyway! I agree that it probably should be reserved for exercises where variables are important :slightly_smiling_face:

**Pull request merged**